src/components/graphQLConnect.ts

```diff
@@ -28,6 +28,27 @@ class GraphQLdb {
     return data.data
   }
 
+  public getReviewData = async (reviewId: number) => {
```

Comment:
Finally, we use a GraphQL query in the back-end! I knew the DBConnect layer would come in handy eventually ;)

Reply:
Yeah, neither was I. But I think the reasoning was that you'd be able to swap an SQL query for a GraphQL one without having to touch any of the Actions that call it.